src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+public class Deadline extends Task {
+
+    protected String by;
+
+    /**
+     * 
+     * @param description the description of the task
```

Comment:
Do check your javadoc formatting. Should add the period at the end of each line.